docs/index.adoc

```diff
@@ -0,0 +1,125 @@
+= WildFly Images
+
+WildFly publishes images to run the application server with different JDK versions.
+The tag of the image identifies the version of WildFly as well as the JDK version in the images.
+
+For each release of WildFly (e.g. `34.0.1.Final`), there are fixed tags for each supported JDK version:
+
+* `quay.io/wildfly/wildfly:34.0.1.Final-jdk17`
+* `quay.io/wildfly/wildfly:34.0.1.Final-jdk21`
+
+There are also floating tags available to pull the _latest release of WildFly on the various JDK_:
+
+* `quay.io/wildfly/wildfly:latest-jdk17`
+* `quay.io/wildfly/wildfly:latest-jdk21`
+
+Finally, there is the `latest` tag that pull the _latest release of WildFly on the latest LTS JDK version_:
+
+* `quay.io/wildfly/wildfly:latest`
+
+---
+**NOTE**
+
+_This floating tag may correspond to a different JDK version in future releases of WildFly images._
+
+Instead of using the `latest` tag, we recommend to use the floating tag with the JDK version mention to guarantee the use of the same JDK version across WildFly releases (e.g. `latest-jdk17`).
+
+---
+
+== Usage
+
+To boot in standalone mode
+
+    podman run -p 8080:8080 quay.io/wildfly/wildfly
+    
+To boot in domain mode
+
+    podman run quay.io/wildfly/wildfly /opt/jboss/wildfly/bin/domain.sh -b 0.0.0.0 -bmanagement 0.0.0.0
+
+== Application deployment
+
+With the WildFly server you can link:https://docs.wildfly.org/34/Admin_Guide.html#application-deployment[deploy your application in multiple ways]
```

Comment:
you don't need to add `link:` prefix in most cases: `https://docs.wildfly.org/34/Admin_Guide.html#application-deployment[deploy your application in multiple ways]` would be rendered fine too.

Reply:
Thanks, fixed.